client/src/js/screens/components/RecipeCard.js

```diff
@@ -6,21 +6,22 @@ import meal1 from './../../../assets/img/meal-1.jpg';
 
 export default class RecipeCard extends React.Component {
```

Comment:
This file has the wrong naming convention for files extensions in React as stated by the airbnb styles guide. 
Please look at this link for more reference 

https://github.com/airbnb/javascript/tree/master/react#naming

Reply:
Thanks for your feedback. I'll rename the components following the appropriate naming conventions.